Objects/obmalloc.c

```diff
@@ -1286,6 +1432,33 @@ _PyObject_Realloc(void *ctx, void *p, size_t nbytes)
     return bp ? bp : p;
 }
 
+
+static void *
+_PyObject_AlignedAlloc(void *ctx, size_t alignment, size_t size)
+{
+    if (alignment <= ALIGNMENT) {
```

Comment:
`alignment <= ALIGNMENT && size <= SMALL_REQUEST_THRESHOLD`

should fix the call chain problem.